aten/src/ATen/core/jit_type_base.h

```diff
@@ -156,7 +156,7 @@ struct TORCH_API Type {
   Type(Type&&) noexcept = default;
   Type& operator=(Type&&) noexcept = default;
 
-  virtual std::string annotation_str_impl(TypePrinter /*printer*/) const {
+  virtual std::string annotation_str_impl(const TypePrinter& /*printer*/) const {
```

Comment:
Nit (for the sake of uniformity)
```suggestion
  virtual std::string annotation_str_impl(C10_UNUSED const TypePrinter& printer) const {
```